src/hud.h

```diff
@@ -61,6 +61,7 @@ enum HudElementType {
 	HUD_ELEM_STATBAR   = 2,
 	HUD_ELEM_INVENTORY = 3,
 	HUD_ELEM_WAYPOINT  = 4,
+	HUD_ELEM_COMPASS   = 6,
```

Comment:
why 6 and not 5 ?

Reply:
5 will be `minimap`.
https://github.com/minetest/minetest/pull/9079/files#diff-036e439b344795cd3c89ffb3153b05d7R64

Reply:
Thanks SmallJocker, that's it.
I was hesitating between having same number in the two PRs then reajust, or try to hide this detail from perspicacious eyes of core devs :)